source/portable/NetworkInterface/STM32Hxx/NetworkInterface.c

```diff
@@ -622,23 +619,23 @@ static BaseType_t xSTM32H_NetworkInterfaceOutput( NetworkInterface_t * pxInterfa
             /* Memory barrier: Make sure that the data written to the packet buffer got written. */
             __DSB();
 
-            /* Get exclusive access to the TX process.
+            /* Get exclusive accces to the TX process.
```

Comment:
I think that `access` was OK with two C's and two S's

Reply:
Thanks Hein, updated in the latest version.